### ISSUE TITLE
cannot know if user confirm or deny upon redirect after calling /endsession

### ISSUE THREAD

**Issue by gianpaolo-tndigit** [scc-digitalhub/AAC#75]
premise: post_logout_redirect_uri contains two functions:

- A: contains actions to be performed in case the user leaves the session
- B: contains actions in case the user changes his mind and cancel the request to endsession

once the API /endsession has been called with id_token_hint and post_logout_redirect_uri AAC asks to confirm or cancel the request.
As soon as the user makes his choice, AAC redirects to the post_logout_redirect_uri link.
The problem is that the user's choice is not passed in the post_logout_redirect_uri so you have no way of knowing if you need to proceed with function A or function B.
![conferma_uscita](https://user-images.githubusercontent.com/100349014/161085800-50f24e31-6f3f-492b-a7c2-6495eb2375ed.GIF)

**Comment by matteo-s** [comment]
I think you are misunderstanding the behavior.
This is oidc rp-initiated logout, and it exposes just one function: signal to the Idp that the RP *has already* logged out the user. Your premise is incorrect.

The flow in plain english:
1. Rp terminates its own local session for user
2. Rp asks Idp to logout user from ISP session
3. If the users agrees to logout from Idp, the Idp will use the front channel or back channel logout to inform every other client of the logout
4. When provided with a valid redirect,the Idp will send the user back to RP

There is nothing in the flow designed to give the RP a feedback on session status at Idp level, and the premise is that the RP *has already logged out the user* so no reason to receive any feedback. This is *only* to inform the Idp, in order to support global session management.
The redirect will be performed regardless of user action.
What you desire (a callback on logout at Idp) may be obtained via back channel logout when we'll support it.
